sparkler-app/src/main/scala/edu/usc/irds/sparkler/pipeline/Crawler.scala

```diff
@@ -59,7 +64,7 @@ class Crawler extends CliTool {
 
   @Option(name = "-o", aliases = Array("--out"),
     usage = "Output path, default is job id")
-  var outputPath: String = ""
+  var outputPath: String = "crawl-segments"
```

Comment:
what is the intention of this change?

There is a statement like this:
```
if (this.outputPath.isEmpty) {
      this.outputPath = jobId
    }
```
so, we set jobId as output path when the path is not specified. It is a good to have feature since, it separates output of each job separately. Now everything goes to one directory.

Reply:
I believe this was unintentional and can be removed.